src/lucky/log_formatters/default_log_formatter.cr

```diff
@@ -0,0 +1,6 @@
+class DefaultLogFormatter < Lucky::LogFormatters::Base
+  
```

Comment:
Could you run `crystal tool format src spec` to remove this and make sure the other stuff is formatted ok?